src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -131,18 +132,19 @@ public EmailWrapper generateFeedbackSubmissionConfirmationEmailForInstructor(
     
     private EmailWrapper generateSubmissionConfirmationEmail(
             CourseAttributes course, FeedbackSessionAttributes session, String submitUrl,
-            String userName, String userEmail, String timeStamp) {
+            String userName, String userEmail, Calendar timeStamp) {
+        TimeHelper.convertToUserTimeZone(timeStamp, session.getTimeZone());
```

Comment:
I thought you changed the method such that it did not modify the original parameter? In which case this line does nothing?
Also, while we're at it, you can change the variable name to `timestamp` (small "s") to make it uniform.